synapse/storage/schema/main/delta/73/06thread_notifications_thread_id_idx.sql

```diff
@@ -13,7 +13,11 @@
  * limitations under the License.
  */
 
--- The columns can now be made non-nullable.
-ALTER TABLE event_push_actions_staging ALTER COLUMN thread_id SET NOT NULL;
-ALTER TABLE event_push_actions ALTER COLUMN thread_id SET NOT NULL;
-ALTER TABLE event_push_summary ALTER COLUMN thread_id SET NOT NULL;
+-- Allow there to be multiple summaries per user/room.
+DROP INDEX IF EXISTS event_push_summary_unique_index;
```

Comment:
Dropping this index is somewhat scary, I don't know how well this will handle rolling back.

Reply:
Well, we can only rollback to v1.68, which I thought we checked was fine?

Reply:
I believe it will work OK with that, yes.